src/main/java/duke/Event.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a specific type of Task with an additional time field.
+ */
+public class Event extends Task {
+    private LocalDate time;
```

Comment:
Since this is saving the date instead of time, maybe you can rename this variable to date instead.

Reply:
Thanks for pointing this out. I have changed the variable name to date in the latest commit!